test/PantosHub.t.sol

```diff
@@ -1253,14 +1496,64 @@ contract PantosHubTest is PantosHubDeployer {
     }
 
     function test_unregisterExternalToken() external {
-        registerTokenAndExternalToken();
+        registerTokenAndExternalToken(PANDAS_TOKEN_OWNER);
         vm.expectEmit();
         emit IPantosRegistry.ExternalTokenUnregistered(
             PANDAS_TOKEN_ADDRESS,
             uint256(otherBlockchain.blockchainId)
         );
 
-        vm.prank(DEPLOYER);
+        vm.prank(PANDAS_TOKEN_OWNER);
+        pantosHubProxy.unregisterExternalToken(
+            PANDAS_TOKEN_ADDRESS,
+            uint256(otherBlockchain.blockchainId)
+        );
+
+        PantosTypes.ExternalTokenRecord
+            memory externalTokenRecord = pantosHubProxy.getExternalTokenRecord(
+                PANDAS_TOKEN_ADDRESS,
+                uint256(otherBlockchain.blockchainId)
+            );
+        assertFalse(externalTokenRecord.active);
+    }
+
+    function test_unregisterExternalTokenBySuperCriticalOps() external {
+        registerTokenAndExternalToken(SUPER_CRITICAL_OPS);
+        vm.expectEmit();
+        emit IPantosRegistry.ExternalTokenUnregistered(
+            PANDAS_TOKEN_ADDRESS,
+            uint256(otherBlockchain.blockchainId)
+        );
+
+        vm.prank(SUPER_CRITICAL_OPS);
+        pantosHubProxy.unregisterExternalToken(
+            PANDAS_TOKEN_ADDRESS,
+            uint256(otherBlockchain.blockchainId)
+        );
+
+        PantosTypes.ExternalTokenRecord
+            memory externalTokenRecord = pantosHubProxy.getExternalTokenRecord(
+                PANDAS_TOKEN_ADDRESS,
+                uint256(otherBlockchain.blockchainId)
+            );
+        assertFalse(externalTokenRecord.active);
+    }
+
+    function test_unregisterExternalTokenBySuperCriticalOpsAndPaused()
```

Comment:
Our naming convention would be `test_unregisterExternalToken_BySuperCriticalOpsAndPaused`.